app/_assets/stylesheets/pages/landing-page.less

```
@@ -14,7 +14,7 @@ header.navbar {
 }
 
 .landing-page {
-  max-width: 1280px;
+  max-width: 100%;
```

```suggestion
  max-width: 1280px;
```
I think you have to leave this as an absolute value, unless you're going to nest it in another div. The landing page looks really awkward in full width on anything wider that 1280 if you set it to 100%:

https://deploy-preview-3022--kongdocs.netlify.app/

If you don't have a monitor, you can test through chrome dev tools.